docs/run-keep.adoc

```diff
@@ -0,0 +1,329 @@
+:toc: macro
+
+= Run Keep
+
+:icons: font
+:numbered:
+toc::[]
+
+== Keep Types?
+
+Here's a bippity boop about keep types.
+
+== Install
+
+Things you will need:
+
+- Keep client executable
+- Keep client configuration file
+- Ethereum account and keyfile
+- KEEP tokens
+
+=== System Recommendations
+
+- It is paramount that Keep nodes remain available to the Keep network. We strongly recommend a stable and redundant internet connection.
+- A connection to a production grade self-hosted or third party Ethereum node deployment.
+- Persistent storage that will survive a VM or container rotation.
+- Machines:
+
+[%header,cols=2*]
+|===
+|Cloud Provider
+|Instance Type
```

Comment:
> Looks like this will need to change. Once this gets settled in the internal env I'll update this section.

Nothing changed here because ecdsa run docs will be in a separate doc all together, and this comment was for ecdsa keeps.